### PR TITLE
Some Docker scaffolding for Arteria images.

### PR DESCRIPTION
**Pull request by johanherman**
Idea is to have a base image with test data and then sub-images that inherits this. See README.md for
more details.

### CODE REVIEW

**Comment by johanherman** [comment]
And https://trello.com/c/54C2ouW7/73-investigate-test-infrastructure-with-docker for more data points; a shared virtual machine for integration tests.

**Comment by johandahlberg** [comment]
Looks good. Maybe @withrocks want to have a look as well, as he know more about the supervisor stuff than me?

File: arteria-docker/services/run [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Example how to run the arteria services docker image
+#
+# We need to mount the source code from the host to e.g. be able to
+# install the services easily.
+#
+# If launc
```
Comment (withrocks) [inline arteria-docker/services/run]: Missing end of sentence. I tend to do this

File: arteria-docker/siswrap/install_services [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+# have been mounted from the host when running this container
+
+set -e
+
+# Start the supervisord
+service supervisord restart
+
+# Install the runfolder package
+cd /arteria/arteria-provisioning/services
+./install-service runfolder root root dev /arteria/arteria-lib/runfolder /arteria/arteria-lib/arteria /opt/runfolder
```
Comment (withrocks) [inline arteria-docker/siswrap/install_services]: This script installs other services besides siswrap, although in the siswrap folder. Should it be so?

**Comment by withrocks** [comment]
Looks cool - look forward to trying this out.

File: arteria-docker/services/run (hunk 1 above)
Comment (johanherman) [inline arteria-docker/services/run]: Doh. :) Old temp files. I've removed this now.

File: arteria-docker/siswrap/install_services (hunk 2 above)
Comment (johanherman) [inline arteria-docker/siswrap/install_services]: I've removed the siswrap Docker files from the arteria-provisioning completely now; keeping it under arteria-lib/siswrap/ instead.

**Pull request merged**